lib/strategies/default.js

```diff
@@ -1,9 +1,24 @@
 'use strict';
 
+var Promise = require('rsvp').Promise;
+
 module.exports = {
   init: function() {},
 
   processString: function(ctx, contents, relativePath) {
-    return ctx.processString(contents, relativePath);
+    var string =  Promise.resolve(ctx.processString(contents, relativePath));
+
+    return string.then(function(value) {
+      var normalizedValue = value;
+
+      if (typeof value === 'string') {
+        normalizedValue = { output: value };
+      }
+
+      return Promise.resolve(ctx.postProcess(normalizedValue, relativePath))
```

Comment:
should likely take the constructor form, so if postProcess itself throws this function is guarenteed to return a promise